impress-org/givewp
# Fix: Remove default value for `continue_button_title` - [GIVE-145]

Resolves #7150  / [GIVE-145]

## Description
This PR removes the static default value for `continue_button_title` in the shortcode args, so that form meta can be used as the default for legacy forms. We still require a default value for v3 forms, so one has been added to the `BlockAttributes` class.

Also I found a duplicate line for `$formUrl` in the `BlockRenderController` that I removed. It was an oversight while merging [Feature: include shortcode & block control support for v3 forms #7082](https://github.com/impress-org/givewp/pull/7082/files#diff-6afdb6343e85db42452a2debf534303d64d06709fd43e3119b0670f89a2fabae)  and [Feature: Embed Donation Form button #7080 ](https://github.com/impress-org/givewp/pull/7080)

## Affects
Legacy Forms, shortcode, block

## Visuals

https://github.com/impress-org/givewp/assets/75056371/e38b77da-c795-4d71-a0a3-bf3e5c0f54d3

## Testing Instructions
- Create a legacy form template
- Select reveal on form display options
- Update Continue button
- View form on frontend & confirm that the button label changes
- Create a shortcode - without explicitly setting continue button
- View form on frontend & confirm that the button label changes
- Retest with a v3 form

## Pre-review Checklist






[GIVE-145]: https://stellarwp.atlassian.net/browse/GIVE-145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
